app/src/main/java/app/revanced/integrations/returnyoutubedislike/ReturnYouTubeDislike.java

```diff
@@ -36,56 +47,105 @@ public enum Vote {
 
     private static Thread _dislikeFetchThread = null;
     private static Thread _votingThread = null;
-    private static Registration registration;
-    private static Voting voting;
     private static CompactDecimalFormat compactNumberFormatter;
 
     static {
-        Context context = ReVancedUtils.getContext();
         isEnabled = SettingsEnum.RYD_ENABLED.getBoolean();
-        if (isEnabled) {
-            registration = new Registration();
-            voting = new Voting(registration);
-        }
-
-        Locale locale = context.getResources().getConfiguration().locale;
-        LogHelper.debug(ReturnYouTubeDislike.class, "locale - " + locale);
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            Context context = ReVancedUtils.getContext();
+            Locale locale = context.getResources().getConfiguration().locale;
+            LogHelper.printDebug(() -> "Locale: " + locale);
             compactNumberFormatter = CompactDecimalFormat.getInstance(
                     locale,
                     CompactDecimalFormat.CompactStyle.SHORT
             );
         }
     }
 
+    private void ReturnYouTubeDislike() {
+    } // only static methods
```

Comment:
This should probably be a constructor.

```suggestion
    private ReturnYouTubeDislike() {
    } // only static methods
```